src/main/java/seedu/bankwithus/Ui.java

```diff
@@ -55,4 +55,13 @@ public String getNextLine() {
     public void closeScanner() {
         this.scanner.close();
     }
+
+    public void viewAccount(String accDetails) {
+        String name = accDetails.split(";")[0];
+        String bal = accDetails.split(";")[1];
```

Comment:
Hahaha, well this works, but i prefer splitting the accDetails into a place holder split, then assigning name and balance after that. Ofc your way works too, no need to change this for now.